### ISSUE TITLE
arm64 is missing from ARCHS

### ISSUE THREAD

**Issue by pshirshov** [Swinject/SwinjectAutoregistration#66]
Swinject itself builds for M1 Macs, but for some reason SwinjectAutoregistration doesn't:

```
SwinjectAutoregistration on  master
❯ grep ARCHS .
./Configurations/Base/Common.xcconfig
11:ARCHS = $(ARCHS_STANDARD)

./Configurations/Mac OS X/Mac-Base.xcconfig
19:VALID_ARCHS = x86_64
```

Could you please fix that?

**Comment by captainOfM** [comment]
i have same issue with new carthage 0.37.0 version and xcframeworks.

**Comment by serg-vinnie** [comment]
same issue

**Comment by serg-vinnie** [comment]
made pull request with fix #68

**Comment by yoichitgy** [comment]
Merged PR #68. Thank you😃
